dist/s_mentions

```diff
@@ -16,11 +16,16 @@ fi
 ticket_id=$(echo "$branch_name" | cut -d "-" -f-2)
 
 # Find the name of the build folder WiredTiger has been compiled in.
```

Comment:
nit: Could we move the declaration of `search_function` up one line so the original comment isn't split? 
We could also pluralise the `cmake_file` variable and relevant comments to make it clear we're considering multiple directories